test/card-test.js

```diff
@@ -3,15 +3,21 @@ var Application = require('spectron').Application;
 var electron = require('electron-prebuilt');
 var assert = require('assert');
 const path = require('path');
+var fs = require('fs');
 var Card = require('../app/Card.js');
+var Canvas = require('../app/Canvas.js');
+var TextEditor = require('../app/textEditor.js')
+var SketchPad = require('../app/sketchPad.js')
+var CodeEditor = require('../app/codeEditor.js')
 
 describe('cards interactions', function () {
   this.timeout(30000);
   var app;
 
   before(function () {
-    app = new Application({
-      path: electron,
+    this.jsdom = require('jsdom-global')()
+    global.$ = global.jQuery = require('jquery');
+      app = new Application({      path: electron,
```

Comment:
For line 20, there isn't really a good reason to shorten the `path:` field onto the same line as the `new Application` declaration; this change adds clutter.